src/utils/api/block/index.js

```diff
@@ -12,11 +14,18 @@ const wsMethods = {
   blocksChange: 'update.block',
 };
 
+const getBlockProps = ({ id, height }) => {
+  if (id) return { id };
+  if (height) return { height };
+  return {};
```

Comment:
If you don't pass anything, if will retrieve the list of latest blocks, the component will pick up the first one and that will result in wrong data on the page. I think if none of the above params was sent, this method should fail.